common/config/babel/.babelrc.js

```diff
@@ -34,7 +34,9 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // dialpad
         'dialpad',
         // rooms
-        'rooms'
+        'rooms',
+        // Participant pane in the `CallComposite`.
+        'call-composite-participant-pane'
```

Comment:
This work is part of the 1-to-n-calling and has a tag already defined in babelrc :)

Reply:
Ah ha, I will retag everything to use  "one-to-n-calling"!